src/libs/antares/study/study.cpp

```diff
@@ -1088,6 +1088,15 @@ void Study::initializeProgressMeter(bool tsGeneratorOnly)
             }
             progression.add(y, Solver::Progression::sectTSGThermal, n);
         }
+        if (TS<timeSeriesShortTermStorage>::IsNeeded(*this, y))
+        {
+            n = parameters.nbTimeSeriesShortTermStorage * areas.size() * 365;
+            if (0 != (timeSeriesShortTermStorage & parameters.timeSeriesToArchive))
+            {
+                n += areas.size();
+            }
+            progression.add(y, Solver::Progression::sectTSShortTermStorage, n);
+        }
```

Comment:
```suggestion
            n = parameters.nbTimeSeriesShortTermStorage * areas.size();
            progression.add(y, Solver::Progression::sectTSShortTermStorage, n);
        }
```

Reply:
i wonder if this code (mine) has a sense, i mean i'll remove nbTimeSeriesShortTermStorage